Mathlib/Data/Bool/Basic.lean

```diff
@@ -2,13 +2,16 @@
 Copyright (c) 2014 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura, Jeremy Avigad
+
+! This file was ported from Lean 3 source module data.bool.basic
+! leanprover-community/mathlib commit c4658a649d216f57e99621708b09dcb3dcccbd23
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
 -/
 import Mathlib.Init.Data.Bool.Lemmas
 import Mathlib.Init.Data.Nat.Lemmas
 import Mathlib.Init.Function
 
-#align_import data.bool.basic from "leanprover-community/mathlib"@"c4658a649d216f57e99621708b09dcb3dcccbd23"
```

Comment:
Please don't convert these to old headers like this; I would guess you did a bad merge at some point?

Reply:
Yeah. I think that's due to some merge.

Reply:
I've unresolved this because the offending change is still here.

Reply:
I see this header is also in master.

Reply:
I see the old header in this file in the master branch 🤔.